### ISSUE TITLE
Scale-down triggered during pod startup delays

### ISSUE THREAD

**Issue by kahirokunn** [actions/actions-runner-controller#3581]
### Checks

- [X] I've already read https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners-with-actions-runner-controller/troubleshooting-actions-runner-controller-errors and I'm sure my issue is not covered in the troubleshooting guide.
- [X] I am using charts that are officially provided

### Controller Version

0.9.1

### Deployment Method

ArgoCD

### Checks

- [X] This isn't a question or user support case (For Q&A and community support, go to [Discussions](https://github.com/actions/actions-runner-controller/discussions)).
- [X] I've read the [Changelog](https://github.com/actions/actions-runner-controller/blob/master/docs/gha-runner-scale-set-controller/README.md#changelog) before submitting this issue and I'm sure it's not due to any recently-introduced backward-incompatible changes

### To Reproduce

```markdown
Provision a new Node or Start the Runner Pod slowly.
```

### Describe the bug

When there is a delay in starting the Runner Pod (e.g., when a new Node is being provisioned), a scale-down is triggered, resulting in the termination of the Runner Pod before it can handle any Jobs. Setting `minRunners` to 1 prevents the scale-down from being triggered. This issue does not occur when using version 0.8.3.

### Describe the expected behavior

The expected behavior is that the Runner Pod should not be scaled down during startup delays, allowing it to handle Jobs once it is ready.

### Additional Context

```yaml
TODO
```

### Controller Logs

```shell
TODO
```

### Runner Pod Logs

```shell
TODO
```

**Comment by nikola-jokic** [comment]
Hey @kahirokunn,

This is a duplicate of https://github.com/actions/actions-runner-controller/issues/3450

**Comment by kahirokunn** [comment]
Oh thx
